counter/.babelrc

```diff
@@ -1,5 +1,10 @@
 {
-  "presets": [
-    ["env", { "modules": false }]
-  ]
+	"presets": [
+		[
```

Comment:
Tabs?

Reply:
Oh! Shit. I totally forgot to reply. Thanks for your contributions. I needed to add some more things to it. Like in the webpack config but that's not why I reverted. The main reason I reverted was because I planned to target the staging branch but I forgot to set it as the base branch.

Reply:
BTW, you won't show up [here](https://github.com/ooade/vuex-examples/graphs/contributors) if i didn't merge.

Looking forward to more PRs from you.